.github/dependabot.yml 

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Dependabot used to have `live` updates but they've been removed with no promise of returning (https://github.community/t/dependabot-update-frequency-live-option/116321).